docs/API.md

```diff
@@ -1541,22 +1542,22 @@ if err != nil {
 ### GetBucketLifecycle(bucketName) (lifecycle string, error)
 Get lifecycle on a bucket or a prefix.
 
-__Parameters__
+__参数__
```

Comment:
These should be removed

Reply:
Thank you very much, I'll correct it as soon as possible.